create_jenkins_job.py

```diff
@@ -124,10 +125,18 @@ def main(argv=None):
         # configure manual triggered job
         job_name = 'ci_' + os_name
         job_data['cmake_build_type'] = 'None'
+        # For the aarch64 job:
+        #  * do it nightly (because it takes a very long time to run); and
+        #  * don't email on test failures (because there are many, likely
+        #    related to qemu)
+        if os_name == 'linux-aarch64':
+            job_data['time_trigger_spec'] = '0 11 * * *'
```

Comment:
Does this trigger the job `ci_linux-aarch64` every night? The nightly triggered build should be a separate job with the `nightly_` prefix while the `ci_` job is used for manually triggering only.